components/driver/rmt.c

```diff
@@ -547,6 +551,8 @@ static void IRAM_ATTR rmt_driver_isr_default(void* arg)
                         p_rmt->tx_len_rem = 0;
                         p_rmt->tx_offset = 0;
                         p_rmt->tx_sub_len = 0;
+                        if(rmt_on_tx_end_handler != NULL)
+                            rmt_on_tx_end_handler(channel);
```

Comment:
We try to use curly braces around single-line if statements, could you please reformat?

Reply:
Sorry, didn't notice that. I'll fix this with the above.